clcache.py

```diff
@@ -793,9 +793,9 @@ def _parseOptionsAndFiles(cmdline):
             'Fp', 'FR', 'doc', 'FA', 'Fe',
             'Fo', 'Fr', 'AI', 'FI', 'FU',
             'D', 'U', 'I', 'Zp',
-            'MP', 'Tc', 'V', 'wd', 'wo',
-            'Yc', 'Yl', 'Tp', 'we',
-            'Yu', 'Zm', 'F', 'Fi',
+            'F', 'Fi', 'MP', 'Tc', 'Tp', 'V',
+            'w1', 'w2', 'w3', 'w4', 'wd', 'we', 'wo', 'Wv',
```

Comment:
A test for this would be nice which verifies that e.g. `['/w3', '1234']` is parsed correctly.